extensions/Worker.Extensions.Http.AspNetCore/src/AspNetMiddleware/ApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Azure.Functions.Worker.Core.Pipeline;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore
+{
+    internal static class ApplicationBuilderExtensions
+    {
+        public static IApplicationBuilder UseSetServiceProviderMiddleware(this IApplicationBuilder builder) =>
```

Comment:
Do we need this methods? Can we inline the middleware registration and remove this class?

Reply:
We really don't -- the one already existed so I added the new one. I'll inline them and remove this class.